nb/projects/pentacular/terrarium/terrarium.md

```diff
@@ -117,59 +133,119 @@ const makeTerrarium = () => (shape) =>
   shape
     .get('terrarium')
     .in()
-    .note(
-      `Terrarium: ${length}x${width}x${height} thickness ${thickness} hinge length ${hingeLength} hole diameter ${holeDiameter}`
-    )
     .each(flat())
     .section()
     .offset(kerf)
-    .pdf('faces', outline().page('pack', { itemsPerPage: 1 }));
+    .note('#### Panels')
+    .note(
```

Comment:
I know I have said it before, but .note reads so much nicer than .md

Reply:
Yeah :) Gradually getting less bad.